docs/api/search-all.json

```diff
@@ -11,6 +17,12 @@
   {
     "description": "Tests if no pipeline is set, it defaults to the default one",
     "download": "/epr/default-pipeline/default-pipeline-0.0.2.tar.gz",
+    "downloads": [
```

Comment:
@jfsiii WDYT about this format?

Reply:
Could we use the type as the key? Unless I am mistaken this should not be possible to have two artifacts of type tar.gz for the same thing?

Reply:
I am putting myself into a user would want to use curl and jq to fetch the artifact.

Reply:
I think there is a future with multiple artifacts for the same time. Lets assume we add quick intro movies to the packages in the future. By default, these would not be served in the package but directly from the CDN. But there should be a url for example `-full.tar.gz` that will also include the movie files. Same for some massive data, by default it would be streamed from the CDN, but it could also be downloaded in the package.

But I'm not happy with calling it `type` and the value `tar.gz` also seems odd. What I think is important is the Kibana can easily pick the right download link. Perhaps we are back at naming the downloads with a key? @jfsiii thoughts?

Reply:
I see your point for movies or other related artifacts.
I would say type is **OK**, we can version the API later on, iteration over perfection.

Reply:
@jfsiii If I get your comment right, you are good with moving forward with what we have in this PR but switch `type: tar.gz` to `type: tar`?

Reply:
@ruflin Correct. Let's follow the `tar` example from https://artifacts-api.elastic.co and we can add `mime_type: "application/gzip"` or any additional info if we need it

Reply:
Adjusted.